presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -101,6 +101,21 @@ public void testExecuteQuery()
         }
     }
 
+    @Test
+    public void testGetMetadata()
+            throws Exception
+    {
+        try (Connection connection = createConnection();
+                PreparedStatement statement = connection.prepareStatement("SELECT 123")) {
+            PrestoResultSetMetaData rsm = (PrestoResultSetMetaData) statement.getMetaData();
+            assertEquals(rsm.getColumnName(1), "_col0");
+            assertEquals(rsm.getCatalogName(1), "");
+            assertEquals(rsm.getSchemaName(1), "");
+            assertEquals(rsm.getTableName(1), "");
```

Comment:
I would suggest to confirm catalog, schema and table names by creating a temporary table. You can refer `testExecuteUpdate` method in the same class.

Reply:
@ebyhr thank you, will do